### ISSUE TITLE
ansible-lint raises AttributeError for malformed import_tasks

### ISSUE THREAD

**Issue by mdaniel** [ansible-community/ansible-lint#707]
# Issue Type
- Bug report

# Ansible and Ansible Lint details

This is as of 6efa16e7 installed via `pip install .`

```console
$ ansible --version
ansible 2.9.6
  config file = None
  configured module search path = ['/usr/share/ansible/plugins/modules']
  ansible python module location = /proj/ansible/ansible-lint/venv/lib/python3.7/site-packages/ansible
  executable location = /proj/ansible/ansible-lint/venv/bin/ansible
  python version = 3.7.7 (default, Mar 11 2020, 11:44:20) [GCC 9.2.1 20191008]

$ ansible-lint --version
ansible-lint 4.2.1.dev63+g6efa16e
```

- ansible installation method: it came from `pip install .`
- ansible-lint installation method: `pip install .`

# Desired Behaviour

```console
$ ansible-lint -vvvv pb.yml 
  File "/proj/ansible/ansible-lint/venv/lib/python3.7/site-packages/ansiblelint/utils.py", line 151, in _playbook_items
    raise RuntimeError("What kind of nonsense is %r" % pb_data)
RuntimeError: What kind of nonsense is ['oops']
```

# Actual Behaviour (Bug report only)

Be aware this is the ansible-lint side of ansible/ansible#68605

`bug.yml`

```yaml
---
- oops
```
`pb.yml`

```yaml
- hosts: all
  gather_facts: no
  tasks:
  - import_tasks: bug.yml
```

```console
$ ansible-lint -vvvv pb.yml 
## snip
    return [item for play in pb_data for item in play.items()]
AttributeError: 'AnsibleUnicode' object has no attribute 'items'
```

**Comment by webknjaz** [comment]
> This is as of [6efa16e](https://github.com/ansible/ansible-lint/commit/6efa16e70f330d5398c7b0240713a700606b0e02)

I believe you're mistaken here. That commit is unrelated to your error in any way.

Also, your error looks like it's a part of a traceback. Please post the complete trace w/o cutting it.

Could you please submit a PR with test cases (for our pytest runner) that demonstrate this issue?

**Comment by mdaniel** [comment]
> I believe you're mistaken here. That commit is unrelated to your error in any way.

My apologies, I only meant that I had tested the most up-to-date code as of that sha; I didn't mean to blame it on that commit. Sorry for the confusion

> Also, your error looks like it's a part of a traceback. Please post the complete trace w/o cutting it.

```
$ ansible-lint -vvvv issue_707.yml
Traceback (most recent call last):
  File "/Users/mdaniel/PycharmProjects/ansible-lint/venv/bin/ansible-lint", line 8, in <module>
    sys.exit(main())
  File "/Users/mdaniel/PycharmProjects/ansible-lint/venv/lib/python3.7/site-packages/ansiblelint/__main__.py", line 185, in main
    matches.extend(runner.run())
  File "/Users/mdaniel/PycharmProjects/ansible-lint/venv/lib/python3.7/site-packages/ansiblelint/__init__.py", line 270, in run
    for child in ansiblelint.utils.find_children(arg, self.playbook_dir):
  File "/Users/mdaniel/PycharmProjects/ansible-lint/venv/lib/python3.7/site-packages/ansiblelint/utils.py", line 164, in find_children
    items = _playbook_items(playbook_ds)
  File "/Users/mdaniel/PycharmProjects/ansible-lint/venv/lib/python3.7/site-packages/ansiblelint/utils.py", line 149, in _playbook_items
    return [item for play in pb_data for item in play.items()]
  File "/Users/mdaniel/PycharmProjects/ansible-lint/venv/lib/python3.7/site-packages/ansiblelint/utils.py", line 149, in <listcomp>
    return [item for play in pb_data for item in play.items()]
AttributeError: 'AnsibleUnicode' object has no attribute 'items'
```
As requested, #720 produces the current error in CI:

> E           AttributeError: 'AnsibleUnicode' object has no attribute 'items'

**Comment by ssbarnea** [comment]
This no longer happens with latest versions which generate a proper error.
